### DISCUSSION TITLE
v10 unsetNodesSelection equivalent for cancelling the selection marquee while keeping the selection

### DISCUSSION THREAD

**Discussion by nousernames2** [xyflow/xyflow#2201] [Q&A] [answered]
I'm trying to clear the marquee selection rectangle from the canvas when a user does a Shift+Drag selection of multiple nodes.

I want to keep the selection active but I don't want the marquee to be seen anymore. In v9 this can be achieved with `unsetNodesSelection` but I cannot find a solution in v10

**Comment by moklick** [comment, marked as answer]
Hey @jcdiprose 

The v10 way is:

```js
const store = useStoreApi();

const unsetSelection = () => {
  store.setState({ nodesSelectionActive: false });
}

// now you can call unsetSelection in an event handler 
```

**Reply by nousernames2** [reply]
Thank you!
